### PR TITLE
6 arm support for instance manager

### PR DESCRIPTION
**Pull request by boknowswiki**
https://github.com/longhorn/longhorn/issues/6

### CODE REVIEW

**Comment by meldafrawi** [comment]
LGTM

File: .drone.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ steps:
 
 - name: build
   pull: default
-  image: rancher/dapper:1.11.2
+  image: rancher/dapper:v0.5.3
```
Comment (joshimoo) [inline .drone.yml:17]: Is this a downgrade or did the versioning scheme change?

Comment (boknowswiki) [reply .drone.yml:17]: Good catch, let me update it. Thanks.

Comment (joshimoo) [reply .drone.yml:17]: No problem, looks like the dapper versioning scheme was changed.
@boknowswiki , @meldafrawi We should make sure to update to the the same version for all of our components.

**Pull request merged**